hypershift-operator/controllers/nodepool/metrics/metrics.go

```diff
@@ -88,11 +92,16 @@ var (
 		countByHClusterMetricHelp,
 		hclusterLabels, nil)
 
-	coresCountByHClusterMetricDesc = prometheus.NewDesc(
-		CoresCountByHClusterMetricName,
-		CoresCountByHClusterMetricHelp,
+	vCpusCountByHClusterMetricDesc = prometheus.NewDesc(
+		VCpusCountByHClusterMetricName,
+		VCpusCountByHClusterMetricHelp,
 		hclusterLabels, nil)
 
+	vCpusComputationErrorByHClusterMetricDesc = prometheus.NewDesc(
+		VCpusComputationErrorByHClusterMetricName,
+		VCpusComputationErrorByHClusterMetricHelp,
+		append(hclusterLabels, "error_reason"), nil)
```

Comment:
(nit) "error" is already in the metric name.
```suggestion
		append(hclusterLabels, "reason"), nil)
```

Reply:
Yeah I wanted to differentiate a bit this label from the other labels which are common to all other metrics.

Reply:
Thanks for pointing that out; proposed change taken into account.